sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 - Improves the performance of the `ServiceBusMessageBatch.tryAdd` method.
   [PR 8772](https://github.com/Azure/azure-sdk-for-js/pull/8772)
+- `Sender` now has an `open()` method to proactively initialize the connection. In addition `ServiceBusClient.createSender()` is no longer an `async` method.
```

Comment:
Nit: Do we need to put this under a breaking changes section since a user can no longer do `createSender().then`?

Reply:
That's a good point, I'll update after the live tests finish.

Reply:
Done.